site/app/views/admin/SqlToolboxView.php

```diff
@@ -7,14 +7,16 @@
 use app\views\AbstractView;
 
 class SqlToolboxView extends AbstractView {
-    public function showToolbox(): string {
+    public function showToolbox($tables): string {
```

Comment:
Should always try to provide some sort of type hint if possible.
```suggestion
    public function showToolbox(array $tables): string {
```